management-service/src/main/java/pl/pwr/zpi/notifications/slack/service/SlackReceiverService.java

```diff
@@ -4,51 +4,91 @@
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.stereotype.Service;
 import pl.pwr.zpi.notifications.common.ConfidentialTextEncoder;
+import pl.pwr.zpi.notifications.discord.dto.UpdateDiscordReceiverRequest;
+import pl.pwr.zpi.notifications.discord.entity.DiscordReceiver;
 import pl.pwr.zpi.notifications.slack.dto.SlackReceiverDTO;
+import pl.pwr.zpi.notifications.slack.dto.UpdateSlackReceiverRequest;
 import pl.pwr.zpi.notifications.slack.entity.SlackReceiver;
 import pl.pwr.zpi.notifications.slack.repository.SlackRepository;
 
 import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 @Service
 @RequiredArgsConstructor
 @Slf4j
 public class SlackReceiverService {
 
-    //    private final SlackNotificationService slackNotificationService;
     private final SlackRepository slackRepository;
     private final ConfidentialTextEncoder confidentialTextEncoder;
 
-    public boolean existsById(Long receiverId) {
-        return slackRepository.existsById(receiverId);
-    }
+    private final String WEBHOOK_URL_REGEX = "https://hooks.slack.com/services/[A-Z0-9]+/[A-Z0-9]+/[a-zA-Z0-9]+";
```

Comment:
maybe we should move that to properties WDYT?

Reply:
Sure, I will move this one and the discord one when cleaning up the draft